modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/context/StoreContext.js

```diff
@@ -19,7 +19,9 @@ const INITIAL_STATE = {
 const ADD_HISTORICAL_WARNING = 'add-historical-warning';
 const ADD_WARNING = 'add-warning';
 
-export const StoreContext = createContext([INITIAL_STATE, () => {}]);
+const noop = () => {};
+
+export const StoreContext = createContext([INITIAL_STATE, noop]);
```

Comment:
In this specific instance you don't benefit from the `noop` trick because `createContext` is called exactly once.